### PR TITLE
Enable AssemblyResolver for Full Framework GenFacadesTasks version

### PR DESCRIPTION
**Pull request by safern**
Fixes: https://github.com/dotnet/buildtools/issues/1443

cc: @weshaggard @mellinoe 

FYI: @jcouv

### CODE REVIEW

**Comment by jcouv** [comment]
Thanks

File: src/Microsoft.DotNet.Build.Tasks/GenFacadesTask.cs [hunk 1]
```diff
@@ -11,7 +15,7 @@ namespace Microsoft.DotNet.Build.Tasks
     /// <summary>
     /// Runs GenFacades In-Proc.
     /// </summary>
-    public sealed class GenFacadesTask : Task
+    public sealed partial class GenFacadesTask : Task
```
Comment (weshaggard) [inline src/Microsoft.DotNet.Build.Tasks/GenFacadesTask.cs]: I think you should just inherit from BuildTask instead.

Comment (weshaggard) [inline src/Microsoft.DotNet.Build.Tasks/GenFacadesTask.cs]: cc @ericstj

Comment (safern) [inline src/Microsoft.DotNet.Build.Tasks/GenFacadesTask.cs]: That would work and would be cleaner.

Comment (safern) [inline src/Microsoft.DotNet.Build.Tasks/GenFacadesTask.cs]: Fixed it. Made GenFacadesTask inherit from BuildTask instead.

**Pull request merged**